app/assets/javascripts/searchform.js

```diff
@@ -10,3 +10,21 @@ $(function() {
     window.location = "/search/" + encoded_query;
   });
 });
+
+$(function () {
+  $('body').on('keydown', '#searchform_input', function (e) {
+    if (e.which === 32 && this.value === '') {
+      return false;
+    }
+  });
+});
+
+function moreChar(textbox) {
```

Comment:
Please change function name to checkInputLength or searchInputLength. Thanks!

Reply:
okay @gauravano 
thanks : )